app/models/user_mute.rb

```diff
@@ -0,0 +1,45 @@
+# == Schema Information
+#
+# Table name: user_mutes
+#
+#  id           :bigint(8)        not null, primary key
+#  creator_id   :bigint(8)        not null
+#  appointee_id :bigint(8)        not null
+#  created_at   :datetime         not null
+#  updated_at   :datetime         not null
+#
+# Indexes
+#
+#  index_user_mutes_on_appointee_id                 (appointee_id)
+#  index_user_mutes_on_creator_id                   (creator_id)
+#  index_user_mutes_on_creator_id_and_appointee_id  (creator_id,appointee_id) UNIQUE
+#
+# Foreign Keys
+#
+#  fk_rails_...  (appointee_id => users.id)
+#  fk_rails_...  (creator_id => users.id)
+#
+class UserMute < ApplicationRecord
+  belongs_to :creator, :class_name => "User"
+  belongs_to :appointee, :class_name => "User"
```

Comment:
I think `appointee` is a pretty odd term to use for the muted user.

I mean I could argue for `mutor` and `mutee` for the creator and person being muted but that might be taking english a bit far.

Maybe `subject` for the person being muted?

On the other side `creator` isn't terrible but maybe `owner` is better? Not sure either really captures the concept well though.

Reply:
> Maybe `subject` for the person being muted?

I like `subject` much better!

I initially used `target` but didn't like it and went with `appointee` 😬 `subject` is better. I'll adjust it.

Reply:
> … maybe `owner` is better?

I went with `creator` b/c `creator` is used for `UserBlock`-records but `owner` is imho also expressive and being used e.g. in the concept of `OAuth`-associated records.

I am not having feelings strong enough towards 1 or the other and am happy to go with `owner`. I'll adjust this.